.travis.yml

```diff
@@ -6,6 +6,7 @@ os:
 
 julia:
   - 1
+  - 1.0
```

Comment:
Let's change these lines into a single `- 1.0` since currently `1` is only an alias for `1.3`.